src/System Application/App/File System/permissions/FileSystemObjects.PermissionSetExt.al

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+permissionset 9452 "File System - Objects"
+{
+    Access = Internal;
+    Assignable = false;
+
+    Permissions =
+        table "File Account" = X,
+        table "File System Connector" = X,
+        table "File System Connector Logo" = X,
+        table "File Account Scenario" = X,
+        table "File Scenario" = X,
+        table "File Account Content" = X,
+        codeunit "File Account" = X,
+        codeunit "File Account Impl." = X,
```

Comment:
you already added InherentPermissions to the Impl. codeunit so there is no need to have it here as well